### ISSUE TITLE
Update CC & MM pop-ups in Sim 

### ISSUE THREAD

**Issue by staceypark** [lzim/teampsd#1046]
**1. Add description:** 
Related to the CC & MM pop-ups for the videos (#748), I noticed that we have outdated language in the pop-ups on the sim.

Our two files at mtl.how/data are now 1) data ui and 2) team data table.
The top text in both CC & MM pop-ups should be updated to say **"Team Data Table at mtl.how/data"** instead of "Team Data Table for Sim UI at mtl.how/data"

![image](https://user-images.githubusercontent.com/31089501/75480534-52d56e00-5956-11ea-9264-41a7d047f96f.png)

 
**2. Click on the right:**  
i. **Pipelines** - prioritize in a TeamPSD (& Workgroup) pipeline  
ii. **Assignee** - assign to an individual team member producing work for the issue  
iii. **Label**- assign workgroup (lead) responsible for tracking the task  
iv. **Projects** - assign to domain-specific TeamPSD tracker (issues, features, document, manuscript, team)  
v. **Milestones** - select monthly sprint milestone within the workgroup   
vi. **Estimate** - Estimate task in complexity points   
vii. **Epics** - assign to team-wide epic and TeamPSD master plan project  
viii. **Releases** - assign to key team release grouped with other relevant issues  

**3. Once you hit "Submit new issue**, underneath the first comment:   
viiii. **Add dependency** - assess for and add dependencies *blocking* & *blocked by* this issue

**Comment by staceypark** [comment]
@jamesmrollins looks like this is done?

**Comment by staceypark** [comment]
Changes have been pushed to TEST and looks correct.
@staceypark updated the documentation with #1043.
@lijenn updated the gifs with #1060 

@jamesmrollins when is this going into PROD?

**Comment by jamesmrollins** [comment]
@staceypark I don't have a release date yet for PROD.  Did you want this change to straight on through to PROD, but leave PLDR and Tutor behind?  Please advise.

**Comment by staceypark** [comment]
Looks like this was moved as part of the MTL 2.0 in PROD
